docs/can-canjs/can-api.md

```diff
@@ -98,11 +97,37 @@ Define custom observable key-value types with [can-observable-object ObservableO
 `ObservableObject` is used to organize the logic of both your [can-stache-element StacheElement] props and your [api#DataModeling Data Models]. The logic
 is expressed as properties and methods.
 
-The following defines a `Todo` type with numerous property behaviors and
-a `toggleComplete` method.
+The following defines a `Todo` type with `toggleComplete` method.
 
 ```js
-import { ObservableObject, ObservableArray, type } from "can";
+import { ObservableObject, type } from "can";
+
+// -------------------------------
+// Define an observable Todo type:
+// -------------------------------
+class Todo extends ObservableObject {
+  static props = {
+    name: string,
+    complete: false
+  };
+
+    // `toggleComplete` is a method
+    toggleComplete() {
+        this.complete = !this.complete;
+    }
+}
+
+// Create a todo instance:
+const todo = new Todo({ name: "Learn Observables" });
+```
+@codepen
+
+<details>
+<summary>Expand to see a larger example with numerous property behaviors:</summary>
```

Comment:
I like how this example is laid out now! But, can you move the ObservableArray example to its own expandable section?